src/config/dynamic-theme-fixes.config

```diff
@@ -6764,6 +6764,8 @@ leetcode.com
 leetcode-cn.com
 
 INVERT
+.node-text__Cnsr
```

Comment:
Where can I find this element?

Reply:
Wherever tree visualizer is available.

For example: https://leetcode.com/problems/binary-tree-inorder-traversal/
Go to Console -> Testcase -> Tree Visualizer

Reply:
Aha.
Il prefer this one in that case:

```CSS

CSS
div[class^="data-structure-viewer"] g[class^="node"] > circle {
    background-color: var(--darkreader-neutral-background) !important;
}
```

Reply:
I tried this out, it does not invert the text inside the circle.
But this works:
```
INVERT
span[class^=node-text]
```

Reply:
I don't think it should invert the text inside the circle. We want to achieve a dark background-circle with white text. At this moment the circle is white as well and should be fixed.

Reply:
In that case, this achieves the desired effect:
```
INVERT
div[class^=data-structure-viewer]
span[class^=node-text]
```

Reply:
No, it doesn't it will invert the whole view and just revert the text, and we don't want to use `INVERT` on unnecessary elements, especially given the fact that in this case we change the `fill` 

```CSS
CSS
div[class^="data-structure-viewer"] g[class^="node"] > circle {
    fill: var(--darkreader-neutral-background) !important;
}
```
This uses the `fill:` property which should work.